Language/Classes/Instance_Methods/Method_noSuchMethod/dynamic_A05_t01.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion For the dynamic semantics, assume that a class C has an
+/// implicitly induced noSuchMethod forwarder named m, with formal type
+/// parameters X1, . . . , Xr, positional formal parameters a1, . . . , ak (some
+/// of which may be optional when n = 0), and named formal parameters with names
+/// x1, . . . , xn
+///
+/// The execution of the body of m creates an instance im of the predefined im
+/// class Invocation such that:
+/// ...
+/// • im.positionalArguments evaluates to an unmodifiable list whose dynamic
+/// type implements List<Object>, containing the same objects as the list
```

Comment:
`Object` would have been changed to `Object?` in the version of the language specification that includes null safety, but I think you have performed that correction already in this test.